recipes/mpir/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "2.7.2" %}
+
+package:
+  name: mpir
+  version: {{ version }}
+
+source:
+  fn: mpir-{{ version }}.tar.bz2
+  url: http://mpir.org/mpir-{{ version }}.tar.bz2
+  md5: b31aa1ebc01b5fedf4ac43277c03feec
+
+build:
+  number: 0
+  skip: true          # [win and py<34]
```

Comment:
Might want to restrict anything beyond Python 3.5. I know it is a bit silly, but this will help us remember what VC is supported here when things change. Could make it `# [win and not (py34 or py35)]`.